### PR TITLE
test: Speed up boots of Fedora, RHEL and CentOS

### PR DESCRIPTION
**Pull request by stefwalter**
We remove the Grub timeout here, as well as prevent eth1
devices from trying DHCP by default on boot.

### CODE REVIEW

**Comment by stefwalter** [comment]
Trying to address slowness issues found in #6334 by @petervo.

File: test/images/scripts/network-ifcfg-eth1 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DEVICE="eth1"
```
Comment (stefwalter) [inline test/images/scripts/network-ifcfg-eth1]: This is missing a ```BOOTPROTO="none"``` line in order to make this take affect properly.

Comment (dperpeet) [inline test/images/scripts/network-ifcfg-eth1]: are you creating new images? Let's block on that then...

Comment (stefwalter) [inline test/images/scripts/network-ifcfg-eth1]: Images are created.

**Pull request merged**